chromebook-setup.sh

```diff
@@ -687,6 +687,14 @@ cmd_deploy_kernel_only()
     cmd_eject_storage
 }
 
+if grep -qi fedora /etc/os-release; then
+  os="fedora"
```

Comment:
Why you need os variable? Isn't pkg_mgr enough? Or I am missing something?

Reply:
Removed